web-api/src/transaction/transaction.helper.ts

```diff
@@ -92,7 +95,7 @@ export const getCardanoBridgingTxFee = async (
 	dto: CreateTransactionDto,
 	skipUtxos: Utxo[] | undefined,
 ): Promise<CardanoTransactionFeeResponseDto> => {
-	const apiUrl = process.env.CARDANO_API_URL || 'http://localhost:40000';
+	const apiUrl = appSettings.cardanoApiUrl || 'http://localhost:40000';
```

Comment:
put default value where settings are loaded (remove this `||` from all places where we set default value from settings like this)